include/framework/runtime.hpp

```diff
@@ -70,6 +70,12 @@ inline constexpr auto LOGGING_CONFIG_NAME = "default_logging.cfg";
 
 inline constexpr auto WWW_DIR = "www";
 
+inline constexpr auto CONTROLLER_PORT = "8849";
+inline constexpr auto MQTT_SERVER = "localhost";
+inline constexpr auto MQTT_PORT = "1883";
+inline constexpr auto MQTT_EVEREST_PREFIX = "everest/";
```

Comment:
i would prefer to skip the trailing slash here

Reply:
maybe even call it MQTT_EVEREST_ROOT_TOPIC

Reply:
Trailing slash is gone here, runtime settings will now check if the prefix ends with a trailing slash, and if not add one.

I've kept the name for now, but that might just be personal preference